examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -37,17 +37,21 @@ export interface ITaskEvents extends IEvent {
 	 */
 	(event: "nameChanged", listener: () => void);
 	/**
-	 * Emitted when the priority has changed to either the incoming value or to the NONE default value.
+	 * Emitted when the priority has changed to either the source value or to the NONE default value.
 	 */
 	(event: "priorityChanged", listener: () => void);
 	/**
-	 * Emitted when the incomingName has changed to either the incoming value or to the NONE default value.
+	 * Emitted when the sourceName has changed to either the source value or to the NONE default value.
 	 */
-	(event: "incomingNameChanged", listener: () => void);
+	(event: "sourceNameChanged", listener: () => void);
 	/**
-	 * Emitted when incomingPriority has changed to either the incoming value or to the NONE default value.
+	 * Emitted when sourcePriority has changed to either the source value or to the NONE default value.
 	 */
-	(event: "incomingPriorityChanged", listener: () => void);
+	(event: "sourcePriorityChanged", listener: () => void);
+	/**
+	 * Emitted when changeType is updated and the conflict UI needs to be displayed.
+	 */
+	(event: "showConflictUI", listener: (value: boolean) => boolean);
```

Comment:
The model probably should not really have a concept of how it is consumed by the view. If this semantically means "I have a conflict that needs to be resolved", I would recommend naming it something along those lines. It's up to the view to understand that this indicates UI needs to be displayed.